packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2416,10 +2434,13 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
             break;
           }
 
-          case RoapScenario.ANSWER:
+          case RoapScenario.ANSWER: {
+            const sdp = this.addSessionConnection(event.roapMessage.sdp);
```

Comment:
I don't think calling SDK should be doing any SDP manipulation. The idea was that all media related things, including SDP handling is managed by internal-mediac-core, so I think these changes should be done in there under a config option that calling would enable.
Also, internal-media-core is already using ts-sdp library for proper parsing of the SDP so it's easier to do any SDP manipulation.